src/collective/lazysizes/tests/test_transformer.py

```diff
@@ -88,3 +95,91 @@ def test_lazyload_tweet_no_script(self):
         element = lxml.html.fromstring(TWEET_NO_SCRIPT)
         # the transformer returns None (skip element)
         self.assertIsNone(self.transformer._lazyload_tweet(element))
+
+    def test_lazyload_availability(self):
```

Comment:
this is already on master now and I split the test to make it simpler:

https://github.com/collective/collective.lazysizes/blob/master/src/collective/lazysizes/tests/test_transformer.py#L54-L65